src/transformers/pipelines/text2text_generation.py

```diff
@@ -181,9 +181,11 @@ def _forward(self, model_inputs, **generate_kwargs):
         elif self.framework == "tf":
             in_b, input_length = tf.shape(model_inputs["input_ids"]).numpy()
 
-        generate_kwargs["min_length"] = generate_kwargs.get("min_length", self.model.config.min_length)
-        generate_kwargs["max_length"] = generate_kwargs.get("max_length", self.model.config.max_length)
-        self.check_inputs(input_length, generate_kwargs["min_length"], generate_kwargs["max_length"])
+        self.check_inputs(
+            input_length,
+            generate_kwargs.get("min_length", self.model.config.min_length),
+            generate_kwargs.get("max_length", self.model.config.max_length),
```

Comment:
If we're not parametrizing from the model config - should we even have it as a default here? 
 ```suggestion
            generate_kwargs["min_length"],
            generate_kwargs["max_length"]
```

Reply:
These lines call a function that may print task-specific suggestions: e.g. in summarization the max length should be smaller than the input length. The user is not required to pass `max_length` (hence the `.get` to obtain the default) -- so we should not accept the suggested change :)

Reply:
Make sense - thanks for explaining!